### PR TITLE
Handle more endpoints

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

File: pyseed/seedclient.py [hunk 1]
```diff
@@ -25,31 +25,43 @@
 # Imports from Standard Library
 import inspect
 
-# Imports from Third Party Modules
-import requests
-
 # Local Imports
+import requests
```
Comment (axelstudios) [inline pyseed/seedclient.py:29]: @nllong This shouldn't be under `Local Imports`, right?

File: pyseed/seedclient.py [hunk 2]
```diff
@@ -194,11 +212,31 @@ def _check_response(self, response, *args, **kwargs):
...
+                elif status_field == 'success':
+                    # continue
+                    error = False
+            elif 'success' in response.json().keys():
+                success_flag = response.json().get('success', None)
+                # For file uploads the response key is 'success'
+                if not success_flag:
+                    error = True
+                elif success_flag:
+                    error = False
```
Comment (axelstudios) [inline pyseed/seedclient.py:229]: `error = not success_flag`

Comment (nllong) [reply pyseed/seedclient.py:229]: haha, so much better.

File: pyseed/seedclient.py [hunk 3]
```diff
@@ -25,31 +25,43 @@
...
         'taxlots': '/api/v3/taxlots/',
+        'upload': '/api/v3/upload/',
         'users': '/api/v3/users/',
+        # POSTs with replaceable keys
+        'import_files_start_save_data_pk': '/api/v3/import_files/PK/start_save_data/',
+        'import_files_start_map_data_pk': '/api/v3/import_files/PK/map/',
+        'import_files_start_matching_pk': '/api/v3/import_files/PK/start_system_matching_and_geocoding/',
+        'org_column_mapping_import_file': 'api/v3/organizations/ORG_ID/column_mappings/',
+        # GETs with replaceable keys
+        'progress': '/api/v3/progress/PROGRESS_KEY/',
```
[the remark this answers is not in the document]
Comment (nllong) [reply pyseed/seedclient.py:63]: `PROGRESS_KEY` since `PK` is PRIMARY KEY.

File: pyseed/seedclient.py (hunk 1 above)
Comment (nllong) [reply pyseed/seedclient.py:29]: This is automated by `isort` so I'm going to leave it.

**Review by axelstudios — APPROVED** [review]

**Pull request merged**